src/client/dbif.cc

```diff
@@ -1006,16 +1006,44 @@ dbif::MethodResultPromise* NCWrapper::handleSetCommentRequest(data::NodeID id,
   return addMethodPromise(qid);
 }
 
-dbif::MethodResultPromise* NCWrapper::handleChangeDataRequest() {
+dbif::MethodResultPromise* NCWrapper::handleChangeDataRequest(data::NodeID id,
```

Comment:
I prefer to not changed that since all methods here have the same issue
and it is temporary code, when new model will be implemented all this things will be removed